### PR TITLE
fixes 35457, 36251, 36496 metering annotation for logging

### PR DESCRIPTION
**Pull request by dian-xh**
changes
- fixes 35457
- fixes 36251
- fixes 36491
- fixes 36496
- added metering annotation for logging

tests
- local deploy happy case validated

### CODE REVIEW

**Comment by dian-xh** [comment]
@kgcarr changes tested locally on aws cluster. please review. thanks!

File: deploy/olm-catalog/ibm-elastic-stack-operator/0.0.1/ibm-elastic-stack-operator.v0.0.1.clusterserviceversion.yaml [hunk 1]
```diff
@@ -15,17 +15,14 @@ metadata:
...
             "image": {
               "pullSecret": {
-                "enabled": "true",
+                "enabled": "false",
                 "name": "infra-registry-key"
               }
             },
             "curator": {
               "image": {
                 "repository": "hyc-cloud-private-edge-docker-local.artifactory.swg-devops.com/ibmcom-amd64/indices-cleaner"
```
Comment (kgcarr) [inline deploy/olm-catalog/ibm-elastic-stack-operator/0.0.1/ibm-elastic-stack-operator.v0.0.1.clusterserviceversion.yaml]: We'll eventually have to change these to quay.io...correct?

Comment (dian-xh) [reply deploy/olm-catalog/ibm-elastic-stack-operator/0.0.1/ibm-elastic-stack-operator.v0.0.1.clusterserviceversion.yaml]: yes, the cicd folks will eventually push workload images to quay. thanks!

**Pull request merged**